PJ05/src/Exceptions/InvalidRequestException.java

```diff
@@ -2,16 +2,13 @@
 
 import Request.Request;
 
-public class InvalidRequestException extends Exception
-{
+public class InvalidRequestException extends RequestFailedException {
     Request request;
 
-    public InvalidRequestException()
-    {
+    public InvalidRequestException() {
     }
 
-    public InvalidRequestException(String message)
-    {
+    public InvalidRequestException(String message) {
         super(message);
```

Comment:
You should make the same constructor for all other subclasses of `RequestFailedException`. They should all take in a `String message` and call `super(message)`.

Note: `String message` is just the error message, not to be confused with `Message message`.